app/views/content/international-qualified-teacher-status.md

```diff
@@ -28,56 +28,50 @@ accordion:
       partial: content/international-qualified-teacher-status/check-your-eligibility
     Find out about fees:
       partial: content/international-qualified-teacher-status/find-out-about-fees
-    Choose your iQTS teacher training provider:
-      partial: content/international-qualified-teacher-status/choose-your-training-provider
-    Apply:
-      partial: content/international-qualified-teacher-status/apply
-    Complete your training:
-      partial: content/international-qualified-teacher-status/complete-your-training
+    Apply to your chosen iQTS teacher training provider:
+      partial: content/international-qualified-teacher-status/apply-to-your-chosen-iqts-teacher-training-provider
+    Gain iQTS and automatic QTS:
+      partial: content/international-qualified-teacher-status/gain-iqts-and-automatic-qts
     Next steps for candidates wishing to teach in England:
       partial: content/international-qualified-teacher-status/next-steps-for-candidates
 ---
 
 ## Introducing iQTS
 
-International qualified teacher status (iQTS) is a new teaching qualification backed by England’s Department for Education (DfE). iQTS is equivalent to [English qualified teacher status](https://www.gov.uk/guidance/qualified-teacher-status-qts) (QTS). You can train where you live and work, with no need to visit the UK. 
+International qualified teacher status (iQTS) is a new teaching qualification backed by the UK government. iQTS meets the same high standards as [English qualified teacher status](https://www.gov.uk/guidance/qualified-teacher-status-qts) (QTS), and leads to the automatic award of QTS. You can train where you live and work, with no need to visit the UK.
 
-## Need to know
+## Who iQTS is for
 
-1. Apply now for courses starting September 2022
-2. Train over the course of an academic year
-3. Learn via remote technology and at a school in your region 
-4. Fees for iQTS vary – check with participating iQTS training providers
-5. Gain a professional teaching qualification recognised as equivalent to QTS
+iQTS is an international teaching qualification designed for UK and non-UK citizens living outside the UK. iQTS means you can gain [qualified teacher status](https://www.gov.uk/guidance/qualified-teacher-status-qts) while working overseas.
```

Comment:
```suggestion
iQTS is an international teaching qualification designed for UK and non-UK citizens living outside the UK. iQTS means you can gain [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) while working overseas.
```

Other links on this page include "(QTS)" in the link - we should be consistent for accessibility reasons